### PR TITLE
Update tempest configuration file for gate environments

### PR DESCRIPTION
**Pull request by pandeyop**
There are changes to tempest configuration file so as to run all tempest tests on the gate environments. One of the restrictions is to boot only from volume in addition to other configurations

### CODE REVIEW

**Comment by pandeyop** [comment]
please test this

**Comment by hkumarmk** [comment]
This PR is depends on jiocloud/puppet-tempest#11

File: hiera/data/common.yaml [hunk 1]
```diff
@@ -857,9 +857,36 @@ tempest::setup_venv: false
 tempest::swift_available: true
 tempest::tempest_clone_path: /home/jenkins/tempest
 tempest::tempest_clone_owner: jenkins
-tempest::allow_tenant_isolation: true
 tempest::volume_storage_protocol: ceph
-
+tempest::extra_config:
+ auth/allow_tenant_isolation:
```
Comment (bodepd) [inline hiera/data/common.yaml]: I'm surprised this hiera config works with only one space and not two. It definitely doesn't follow the conventions of the rest of this file.

**Comment by hkumarmk** [comment]
please test this

File: Puppetfile [hunk 2]
```diff
@@ -34,8 +34,8 @@ mod 'puppetlabs/vscrepo',
   :ref => '1.2.0'
 
 mod 'stackforge/tempest',
-  :git => "#{base_url}/jiocloud/puppet-tempest",
-  :ref => 'master'
+  :git => "#{base_url}/hkumarmk/puppet-tempest",
```
Comment (bodepd) [inline Puppetfile]: this has been merged.

**Comment by pandeyop** [comment]
please test this

**Pull request merged**